autogen/main/firewall.tf.tmpl

```diff
@@ -81,6 +81,7 @@ resource "google_compute_firewall" "master_webhooks" {
   direction   = "INGRESS"
 
   source_ranges = [local.cluster_endpoint_for_nodes]
+  source_tags   = [""]
```

Comment:
CI seems to be failing due to this. IIRC we added this due https://github.com/hashicorp/terraform-provider-google/issues/10494. Maybe we should do `source_tags   = []` as a workaround

```
       Error: Error creating Firewall: googleapi: Error 400: Invalid value for field 'resource.sourceTags[0]': ''. Must be a match of regex '(?:[a-z](?:[-a-z0-9]{0,61}[a-z0-9])?)', invalid
       
         with module.example.module.gke.google_compute_firewall.master_webhooks[0],
         on ../../../firewall.tf line 63, in resource "google_compute_firewall" "master_webhooks":
         63: resource "google_compute_firewall" "master_webhooks" {
```

Reply:
I thought the "correct" (?) fix for that was covered by this:

https://github.com/GoogleCloudPlatform/magic-modules/pull/5526

so we may still see the CI failing until that (or something better) is merged.

On a more personal note, I left my previous employer at the end of last year, so it may be hard for me to take this much further, as the CLA etc. was signed with that email. I'm in touch with a former coworker who I'm going to try and persuade to finish this off for me; I'll let you know how that goes.

Reply:
> On a more personal note, I left my previous employer at the end of last year, so it may be hard for me to take this much further, as the CLA etc. was signed with that email. I'm in touch with a former coworker who I'm going to try and persuade to finish this off for me; I'll let you know how that goes.

Thanks, we can probably follow through if necessary as well.